src/rustdoc-json-types/lib.rs

```diff
@@ -531,13 +531,15 @@ pub enum ItemKind {
     /// Corresponds to either `ItemEnum::Macro(_)`
     /// or `ItemEnum::ProcMacro(ProcMacro { kind: MacroKind::Bang })`
     Macro,
-    /// A procedural macro attribute.
+    /// A macro attribute.
```

Comment:
I noticed `procedural` was deleted here, but was not deleted below in the doc comment for `ProcDerive`. I found that surprising, is that intentional?